README.md

```diff
@@ -5,7 +5,7 @@
 
 ## What are we working towards?
 
-Up until now, the data on our templates has been pretty one-sided. It's sourced from the database, and we render the fields from one or many returned records on the template. Often times, however, the template and the database are not so tightly coupled. There's actually no rule saying that all template data has to come from the database
+Up until now, the data on our templates has been pretty one-sided. It's sourced from the database, and we render the fields from one or many returned records on the template. However, the template and the database are not so tightly coupled; There's no rule saying that all template data has to come from the database
```

Comment:
I think "There's" should be "there's" if following a semi colon

Reply:
Yeah I think you're right.